tests_additional/real_exchanges/test_huobi.py

```diff
@@ -51,8 +51,8 @@ async def test_get_market_status(self):
             assert market_status
             assert market_status[Ecmsc.SYMBOL.value] in (self.SYMBOL, self.SYMBOL_2)
             assert market_status[Ecmsc.PRECISION.value]
-            assert market_status[Ecmsc.PRECISION.value][Ecmsc.PRECISION_AMOUNT.value] >= 1
-            assert market_status[Ecmsc.PRECISION.value][Ecmsc.PRECISION_PRICE.value] >= 1
+            assert 1e-06 <= market_status[Ecmsc.PRECISION.value][Ecmsc.PRECISION_AMOUNT.value] <= 1
+            assert 1e-06 <= market_status[Ecmsc.PRECISION.value][Ecmsc.PRECISION_PRICE.value] <= 1
```

Comment:
this means we will have to update the huobi tentacle to add a market_status fix

Reply:
Yes

Reply:
https://github.com/Drakkar-Software/OctoBot-Tentacles/pull/631